exec/java-exec/src/main/java/org/apache/drill/exec/expr/stat/ParquetPredicates.java

```diff
@@ -66,6 +66,27 @@ public ParquetBooleanPredicate(String name, List<LogicalExpression> args, Expres
     }
   }
 
+  public static abstract class ParquetIsPredicate extends LogicalExpressionBase implements ParquetFilterPredicate {
```

Comment:
`ParquetPredicates` class becomes too overloaded. Please split it into two: `ParquetComparisionPredicates` and `ParquetIsPredicate`.